pkgs/misc/cups/default.nix

```diff
@@ -12,15 +12,15 @@ stdenv.mkDerivation rec {
   pname = "cups";
 
   # After 2.2.6, CUPS requires headers only available in macOS 10.12+
-  version = if stdenv.isDarwin then "2.2.6" else "2.2.11";
```

Comment:
Shouldn't we start marking cups as insecure on darwin?

Reply:
That seems reasonable to me, @NixOS/darwin-maintainers ?